startup.sh

```diff
@@ -19,4 +32,7 @@ pushd "${PN_PROJECT_DIR}/pki"
     "${PKI_OUTPUT_DIR}"
 popd
 
-docker-compose up $@
+docker-compose up $@ -d
+
+wait_for "Metadata" localhost:5000/Metadata
```

Comment:
This is actually waiting for the stub-connector service, rather than the metadata service (which is on 7000). Is that right? Might be worth renaming if so.

Reply:
It's the metadata for the gateway and translator services. Once the healthchecks are added to the other services then I can do the same here with them and get rid of the sleep command pre-commit.

Reply:
Cool. If this check is just to ensure that the service is up and running, it might be worth changing to `wait_for "stub-connector" localhost:5000/Metadata` though.

Reply:
Yeah this is waiting for stub connector to come up so it should probably be renamed.

Reply:
I'll clean this up once the healtchecks are implemented